From 5abd95b65fe5ec24749164338a9718ecce19e240 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Fri, 28 Feb 2020 10:29:58 +0100
Subject: [PATCH] libvnc{client,server}.pc.cmakein: remove zlib

Remove zlib from Requires.private as libvnc can be built without zlib
thanks to WITH_LIB, zlib will be added to Libs.private thanks to
PRIVATE_LIBS

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/LibVNC/libvncserver/pull/377]
---
 libvncclient.pc.cmakein | 2 +-
 libvncserver.pc.cmakein | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libvncclient.pc.cmakein b/libvncclient.pc.cmakein
index 336e73f..ceeda39 100644
--- a/libvncclient.pc.cmakein
+++ b/libvncclient.pc.cmakein
@@ -7,7 +7,7 @@ Name: LibVNCClient
 Description: A library for easy implementation of a VNC client.
 Version: @LibVNCServer_VERSION@
 Requires:
-Requires.private: zlib
+Requires.private:
 Libs: -L${libdir} -lvncclient
 Libs.private: @PRIVATE_LIBS@
 Cflags: -I${includedir}
diff --git a/libvncserver.pc.cmakein b/libvncserver.pc.cmakein
index dfcec9d..33ec668 100644
--- a/libvncserver.pc.cmakein
+++ b/libvncserver.pc.cmakein
@@ -7,7 +7,7 @@ Name: LibVNCServer
 Description: A library for easy implementation of a VNC server.
 Version: @LibVNCServer_VERSION@
 Requires:
-Requires.private: zlib
+Requires.private:
 Libs: -L${libdir} -lvncserver
 Libs.private: @PRIVATE_LIBS@
 Cflags: -I${includedir}
-- 
2.25.0

